python/looker_sdk/rtl/api_settings.py

```diff
@@ -49,49 +49,48 @@ def configure(
         instantiate ApiSettings.
 
         ENV variables map like this:
+            LOOKER_API_VERSION -> api_version
             LOOKER_BASE_URL -> base_url
             LOOKER_CLIENT_ID -> client_id
             LOOKER_CLIENT_SECRET -> client_secret
-            LOOKER_EMBED_SECRET -> embed_secret
-
         """
-        cfg_parser = cp.ConfigParser()
-        try:
-            cfg_parser.read_file(open(filename))
-        except FileNotFoundError:
-            config_data: Dict[str, str] = {}
-        else:
-            # If section is not specified, use first section in file
-            section = section or cfg_parser.sections()[0]
-            config_data = dict(cfg_parser[section])
-
-        env_base_url = cast(str, os.getenv("LOOKER_BASE_URL"))
-        if env_base_url:
-            config_data["base_url"] = env_base_url
-
-        env_client_id = cast(str, os.getenv("LOOKER_CLIENT_ID"))
-        if env_client_id:
-            config_data["client_id"] = env_client_id
 
-        env_client_secret = cast(str, os.getenv("LOOKER_CLIENT_SECRET"))
-        if env_client_secret:
-            config_data["client_secret"] = env_client_secret
-
-        env_embed_secret = cast(str, os.getenv("LOOKER_EMBED_SECRET"))
-        if env_embed_secret:
-            config_data["embed_secret"] = env_embed_secret
+        config_data = cls.read_ini(filename, section)
 
         api_version = cast(str, os.getenv("LOOKER_API_VERSION"))
         if api_version:
             config_data["api_version"] = api_version
```

Comment:
Is there an immediate if/ternary operator we could use for cascading the defaults so this is one statement? e.g.

```python
api_version = cast(str, os.getenv("LOOKER_API_VERSION") or "3.1"
```
for example

Reply:
`api_version = cast(str, os.getenv("LOOKER_API_VERSION") or config_data.get("api_version")` works actually. I think the main reason I did it this way was because `black` still splits the expression into multiple lines because the max line length gets exceeded

Reply:
gotcha. If it's a battle with the code formatting tools, never mind :)